Sources/PackageDescription/PackageDescription.swift

```diff
@@ -456,6 +458,19 @@ public enum SystemPackageProvider {
         return ._aptItem(packages)
       #endif
     }
+
+    /// Declare the list of installable packages using the yum package
+    /// manager on RHEL/CentOS to create a system package provider instance.
+    ///
+    /// - Parameters:
+    ///     - packages: The list of package names.
+    public static func yum(_ packages: [String]) -> SystemPackageProvider {
```

Comment:
This should be marked using package description availability: `@available(_PackageDescription, introduced: 999)`